3기/HyeonkyeongSeo/2주차/css/style.css

```
@@ -0,0 +1,510 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+@font-face {
+    font-family: 'Pretendard-Light';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Light.woff') format('woff');
+    font-weight: 400;
+    font-style: normal;
+}
+
+
+*, ::before, ::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+    font-family: 'Inter', 'Pretendard-Light', sans-serif;
+  font-size: 14px;
+    color: #2D3436;
+}
+
+
+html, body{
+  line-height: 160%;
+}
+
+a{
+    text-decoration: none;
+    color: inherit;
+}
+
+li{
+    list-style: none;
+}
+
+#l_wrapper{
+    width: 100%;
+    max-width: 100%;
+    height: 100%;
+    min-height: 100vh;
+    display: flex;
+    flex-flow: column;
+    justify-content: space-between;
+    background-color: rgba(217,217,217,0.3);
+}
+
+#wrap{
+    max-width: 1200px;
+    min-width: 320px;
+    width: 90%;
+    margin: 0 auto;  
+    display: flex;
+    position: relative;
+}
+
+
+
+/* 모바일 */
+/** 헤더영역 **/
+#header{
+    width: 100%;
+    background-color: white;
+    position: sticky;
+    padding: 15px 0;
+    z-index: 1000;
+}
+
+h1{
+    font-size: 12px;
+    display: block;
+    width: 15%;
+}
+
+#navigation{
+    width: 100%;
+    height: 100vh;
+    position: absolute;
+    top: 0;
+    right: 0;
+    overflow: hidden;
+    
+}
+#navigation.on{
+    background-color: rgba(45,52,54,0.8);   
+}
+
+#gnb{
+    position: absolute;
+    width: 250px;
+    height: 100vh;
+    padding-top: 100px;
+    background-color: white;
+/*    right: 0;*/
+    right: -250px;
+/*    display: none;*/
+}
+#gnb.on{
+    right: 0;
+/*    display: block;*/
+}
+#gnb>li>a{
+    display: block;
+    width: 100%;
+    font-size: 14px;
+    padding: 10px 30px;
+    border-bottom: 1px solid #d8d8d8;
+}
+#gnb>li>a:hover{
+    color: #E0749D;
+}
+#gnb>li.g_active>a:hover{
+    color: #2D3436;
+}
+
+#gnb>li.g_active{
+    font-weight: 900;
+}
+
+#nav_btn{
+    border: 0;
+    text-indent: -9999px;
+    width: 15px;
+    cursor: pointer;
+    position: absolute;
+    top: 3px;
+    right: 5%;
+    z-index: 100;
+    background-color: white;
+}
+
+#nav_btn:hover span{
+    background-color: #E0749D;
+}
+
+#nav_btn span{
+    display: block;
+    width: 100%;
+    height: 2px;
+    background-color: #2D3436;
+}
+
+#nav_btn span:nth-child(2){
+    margin: 3px 0;
+}
+
+#nav_btn.on{
+    width: 19px;
+}
+
+#nav_btn.on span:first-child{
+    transform: translateX(1px) translateY(4px) rotate(45deg);
+}
+
+#nav_btn.on span:nth-child(2){
+    display: none;
+}
+
+#nav_btn.on span:last-child{
+    transform: translateX(1.1px) translateY(2.5px) rotate(-45deg);
+}
+
+
+
+/** 메인영역 **/
+
+#main{
+    width: 100%;
+    min-height: 100%;
+    padding: 100px 0;
+}
```

```suggestion
#main{
    flex: 1 1 auto;
    width: 100%;
    min-height: 100%;
    padding: 100px 0;
}
```

부모가 flexbox이기 때문에 자식 요소들에게 flex-grow, flex-shrink, flex-basis 속성을 넣는걸 잊지말아주세요!
`min-height: 100%`가 들어가 있다곤 하지만, %는 부모의 상대단위이기 때문에 결국 100%만큼 차지 않을 수 있어요!

---

그리고 #main에 공통으로 추가된 padding 값으로 인해서 About페이지의 배경 이미지가 화면에 꽉차지 않게 되고있어요,
이렇기 때문에 "레이아웃을 구성"하는 태그에 디자인적인 외형을 넣으시기 보다,

디자인적인 외형이 필요한 요소, `<div class="about">`, `<div class="timeline">` 등에서 스타일을 처리하시는게 좋아요.